src/main/java/enums/Currency.java

```diff
@@ -8,9 +8,18 @@
 @ToString
 @RequiredArgsConstructor
 public enum Currency {
-    EUR("€"),
-    GBP("£"),
-    USD("$");
+	EUR("€"),
+	GBP("£"),
+	USD("$");
 
-    public final String currencySymbol;
+	public final String currencySymbol;
+
+	public static Currency fromSymbol(String symbol) {
+		for (Currency currency : values()) {
```

Comment:
you can use var currency : values()

Reply:
Why the additional indentations?

Reply:
Done
```java
	public static Currency fromSymbol(String symbol) {
		for (var currency : values()) {
			if (currency.getCurrencySymbol().equals(symbol)) {
				return currency;
			}
		}
		throw new IllegalArgumentException("No enum constant for symbol: " + symbol);
	}
```

Reply:
@jakub-krzysiek You are right. The coe formatting has been improved. Some of the indentation is not so visible in my IDE. Thanks.